itou/templates/job_seekers_views/details.html

```diff
@@ -54,7 +54,7 @@ <h1>Candidat : {{ job_seeker.get_full_name|mask_unless:can_view_personal_inform
 {% endblock %}
 
 {% block title_extra %}
-    {% include "job_seekers_views/includes/nav_tabs.html" with job_seeker=job_seeker active_nav_tab="details" only %}
+    {% include "job_seekers_views/includes/nav_tabs.html" with job_seeker=job_seeker active_nav_tab="details" back_url=back_url only %}
```

Comment:
Est-ce qu'on ne voudrait pas plutôt 
```suggestion
    {% include "job_seekers_views/includes/nav_tabs.html" with job_seeker=job_seeker active_nav_tab="details" back_url=request.get_full_path only %}
```
?